neptune/experiments.py

```diff
@@ -537,6 +547,9 @@ def download_artifact(self, filename, destination_dir):
 
                 experiment.download_artifact('forest_results.pkl', '/home/user/files/')
         """
+        if destination_dir is None:
+            destination_dir = "."
+
         path = "/{exp_id}/output/{file}".format(exp_id=self.id, file=filename)
         destination_path = "{dir}/{file}".format(dir=destination_dir, file=filename)
```

Comment:
This won't work on Windows. Use os.path for both building `destination_path` and splitting `filename`